vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder.go

```diff
@@ -49,7 +49,13 @@ import (
 	resourceclient "k8s.io/metrics/pkg/client/clientset/versioned/typed/metrics/v1beta1"
 )
 
-const defaultResyncPeriod time.Duration = 10 * time.Minute
+const (
+	scaleCacheLoopPeriod         time.Duration = time.Minute
```

Comment:
If we're refreshing every minute, for the usecase where there is one pod per controller, won't we have the same scalability as we have now ?
i.e. currently, if you have 600 pods, one custom controller for each, in the current solution this eats up 600 qpm = 10 qps
with the caching configured like in this PR we still take up  600 qpm = 10 qps

Reply:
Cache entry lives for `scaleCacheEntryFreshnessTime` + jitter (15 minutes on average). Having short `scaleCacheLoopPeriod` is how often we check which entries need a refresh (so it should be shorter than `scaleCacheEntryFreshnessTime` or we'll just refresh everything every loop).

Reply:
Ah of course you're right, sorry